api/src/main/resources/mandrill_api.yaml

```diff
@@ -0,0 +1,110 @@
+#A subset of the Mandrill Api
```

Comment:
Assuming you copied this file from somewhere, could you please put a link here to that location?

Reply:
Actually didn't

Reply:
OK. A link to their API docs would be useful in any case.

Reply:
This is their current doc for this API call: https://mandrillapp.com/api/docs/messages.JSON.html